signer/src/api.rs

```diff
@@ -99,14 +99,16 @@ impl TryFrom<ExecParamsAPI> for ExecParams {
             && exec_constructor.code_cid != "fil/4/multisig"
             && exec_constructor.code_cid != "fil/5/multisig"
             && exec_constructor.code_cid != "fil/6/multisig"
+            && exec_constructor.code_cid != "fil/7/multisig"
```

Comment:
is it time to make this a loop? 😂 
No need to make the change yet.. this is planned to change a lot in the next release anyway

Reply:
Using regexp ?